invokeai/backend/invoke_ai_web_server.py

```diff
@@ -486,6 +485,27 @@ def merge_diffusers_models(model_merge_info: dict):
             except Exception as e:
                 self.handle_exceptions(e)
 
+        @socketio.on('getLoraModels')
+        def get_lora_models():
+            try:
+                lora_path = global_lora_models_dir()
+                lora_folder_ckpt = Path(lora_path).glob("**/*.ckpt")
+                lora_folder_safetensors = Path(lora_path).glob("**/*.safetensors")
```

Comment:
Found some loras on civitai with `.pt` extension instead of `.ckpt`.
Maybe scan files with this extension too?

Reply:
Fixed. Thanks for the suggestion.